.changeset/healthy-stingrays-count.md

```diff
@@ -6,7 +6,7 @@ The new semantic typography components make it clear when each should be used, a
 
 #### Headline
 
-Renamed the `Heading` component to `Headline` and mapped the styles to the new ones. These changes can be codemodded (🤖 _component-names-typography_).
+Renamed the `Heading` component to `Headline` and mapped the styles to the new ones. These changes can be codemodded (🤖 _component-names-v3_).
```

Comment:
Note: I bundled all component name changes into a `component-names-v3` codemod to avoid duplicating the code (just like we did in v2). It now includes the typography component renames as well as the NotificationBanner rename